examples/operator/templates/deployment.yaml

```diff
@@ -56,6 +56,20 @@ spec:
           value: {{ .Values.controllerManager.manager.var2 }}
         - name: VAR3_MY_ENV
           value: {{ .Values.controllerManager.manager.var3MyEnv }}
+        - name: VAR4
+          valueFrom:
+            configMapKeyRef:
+              key: VAR4
+              name: my-operator-configmap-vars
```

Comment:
configmap reference should also be templated if configmap is in the same chart. Please check out `secretKeyRef` sample

Reply:
done, sorry I missed that there was no configmap in the customize output, I just added one